src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/GeneratorTests.Baselines.cs

```diff
@@ -965,5 +965,150 @@ public abstract class AbstractType_CannotInit
+
+                    public myMethodDelegate? SkipProp42 { get; set; }
+                    public Container<myMethodDelegate?> SkipProp43 { get; set; }
+                    public Container<Dictionary<string, myMethodDelegate?>> SkipProp44 { get; set; }
+
+                    public IntPtr? SkipProp45 { get; set; }
+                    public Container<IntPtr?> SkipProp46 { get; set; }
+                    public Container<Dictionary<string, IntPtr?>> SkipProp47 { get; set; }
+
+                    public UIntPtr? SkipProp48 { get; set; }
+                    public Container<UIntPtr?> SkipProp49 { get; set; }
+                    public Container<Dictionary<string, UIntPtr?>> SkipProp50 { get; set; }
+
+                    public SerializationInfo? SkipProp51 { get; set; }
+                    public Container<SerializationInfo?> SkipProp52 { get; set; }
+                    public Container<Dictionary<string, SerializationInfo?>> SkipProp53 { get; set; }
+
+                    public MethodInfo? SkipProp54 { get; set; }
+                    public Container<MethodInfo?> SkipProp55 { get; set; }
+                    public Container<Dictionary<string, MethodInfo?>> SkipProp56 { get; set; }
+
+                    public ConstructorInfo? SkipProp57 { get; set; }
+                    public Container<ConstructorInfo?> SkipProp58 { get; set; }
+                    public Container<Dictionary<string, ConstructorInfo?>> SkipProp59 { get; set; }
+
+                    public string[,]? SkipProp60 { get; set; }
+                    public Container<string[,]?> SkipProp61 { get; set; }
+                    public Container<Dictionary<string, string[,]?>> SkipProp62 { get; set; }
+
+                    public Func<string>?[] SkipProp63 { get; set; }
+                    public Action<string>?[] SkipProp64 { get; set; }
+                    public myMethodDelegate?[] SkipProp65 { get; set; }
+                    public MethodInfo?[] SkipProp66 { get; set; }
+                    public ConstructorInfo?[] SkipProp67 { get; set; }
+                    public SerializationInfo?[] SkipProp68 { get; set; }
+                    public IntPtr?[] SkipProp69 { get; set; }
+                    public UIntPtr?[] SkipProp70 { get; set; }
+
+                    public delegate string myMethodDelegate( int myInt );
+                }
+
+                public class Container<T>
+                {
+                    public Container() {}
+                    public T? Value { get; set; }
+                }
+                """;
+
+            ConfigBindingGenRunResult result = await VerifyAgainstBaselineUsingFile(
+                "UnsupportedTypes.generated.txt",
+                source);
+
+            Assert.Empty(result.Diagnostics);
+            Assert.True(result.GeneratedSource.HasValue);
+            Assert.DoesNotContain(result.GeneratedSource.Value.SourceText.ToString(), "SkipProp");
```

Comment:
Could we also assert no occurrence of the unsupported types, not the types mentioned in generic parameters for any omitted member?

Reply:
Sometimes we generate code using such types (not for binding) here is examples:

```C#

public static int ParseInt(string value, Func<string?> getPath)

public static BinderOptions? GetBinderOptions(Action<BinderOptions>? configureOptions)

```